src/paths/user/account.yaml

```
@@ -0,0 +1,19 @@
+get:
+  tags:
+    - user
+  description: "Show the account details of the signed-in user"
```

```suggestion
  description: "Show the account details of a user"
```
The user specified might not be the one doing the request.